.github/workflows/publish_pytorch_dev_docker.yml

```diff
@@ -1,30 +1,35 @@
 name: Publish PyTorch Dev Dockers
+
 on:
   workflow_dispatch:
   schedule:
-    - cron: "0 2 * * *" # Runs nightly at 2 AM UTC
+    - cron: "0 2 * * *" # Runs Nightly at 2 AM UTC
+
 jobs:
   build-and-push-image:
     strategy:
       fail-fast: false
       matrix:
         targets:
           - amdgpu_target: "gfx942"
-          - amdgpu_target: "gfx1100"
-          - amdgpu_target: "gfx1201"
 
     name: ${{ matrix.targets.amdgpu_target }}::Build
     runs-on: azure-linux-scale-rocm
+    outputs:
+      gfx942_digest: ${{ steps.export.outputs.gfx942 }}
```

Comment:
Variable names should be more generic and not have a single specific arch within them as you want to be able to run these steps also for other GPU architectures.